arwen/contexts/metering.go

```diff
@@ -194,8 +198,25 @@ func (context *meteringContext) deductInitialGas(
 ) error {
 	input := context.host.Runtime().GetVMInput()
 	codeLength := uint64(len(code))
-	codeCost := codeLength * costPerByte
-	initialCost := baseCost + codeCost
+	codeCost, err := math.MulUint64WithErr(codeLength, costPerByte)
```

Comment:
I think it's safe to use `math.MulUint64()` directly, without `Err`, making the code cleaner. The condition at line 221 would prevent using `MaxUint64` gas.

Reply:
Done.